<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>455</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Concesionaria</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_ve">
     <attribute name="title">
      <string>Vehículos</string>
     </attribute>
     <widget class="QTableWidget" name="tabla_ve">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>611</width>
        <height>231</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Modelo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Año</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>611</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="le_filtro_ve">
         <property name="placeholderText">
          <string>¿Qué desea buscar?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nuevo_ve">
         <property name="text">
          <string>Nuevo Vehiculo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_editar_ve">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e8d823;&quot;&gt;Presiona para editar una fila&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Editar Vehiculo</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_eliminar_ve">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#e8d823;&quot;&gt;Presione para eliminar una fila&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Eliminar Vehiculo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ma">
     <attribute name="title">
      <string>Marcas</string>
     </attribute>
     <widget class="QTableWidget" name="tabla_ma">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>611</width>
        <height>231</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pais</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total de Autos</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>611</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="le_filtro_ma">
         <property name="placeholderText">
          <string>¿Qué desea buscar?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nuevo_ma">
         <property name="text">
          <string>Nueva Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_editar_ma">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Editar Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_eliminar_ma">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Eliminar Marca</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
